package.json

```diff
@@ -76,7 +76,7 @@
     "babel-preset-es2015": "6.22.0",
     "babel-preset-react": "6.22.0",
     "bowser": "1.9.4",
-    "chromedriver": "92.0.1",
+    "chromedriver": "94.0.0",
```

Comment:
Should this be in this PR? No strong preference, but seems unrelated to the rest of the tests and something we'd likely want to keep even if the new tests turn out to be flaky or something.

Reply:
I could do it in its own, but usually (and in this case in particular) it needs to be updated because you just can't run the tests without updating Chromderiver.  In the past we've just updated chromedriver in whatever PR is updating tests.  

It is in its own Commit, though, so it should be separable if it needs to be.